target/linux/ramips/image/mt7621.mk

```diff
@@ -1990,6 +1990,24 @@ define Device/netgear_wax202
 endef
 TARGET_DEVICES += netgear_wax202
 
+define Device/netgear_wax214v2
+  $(Device/nand)
+  DEVICE_VENDOR := NETGEAR
+  DEVICE_MODEL := WAX214v2
+  DEVICE_PACKAGES := kmod-mt7915-firmware -uboot-envtools
```

Comment:
Please do not remove `uboot-envtools`. You defined the u-boot settings.

Reply:
Okay, should I also remove `read-only` from the Config partition?

Reply:
if setting uboot environment variables works, make it writable too please.

Reply:
Okay, done